dashboard/app/models/sections/section.rb

```diff
@@ -30,6 +30,18 @@
 require 'cdo/safe_names'
 
 class Section < ActiveRecord::Base
```

Comment:
For this specific case:
```
Word.create! login_type: 'picture'
=> Invalid single-table inheritance type: Picture is not a subclass of Word
```
But I think you're right that we should make `Section.new` throw, and require you to create the specific class you want.  Right now `Section.new` creates an "email" section by default:
```
Section.new
=> #<Email id: nil, login_type: "email", ... >
```